lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -86,6 +88,16 @@ def add_payment_method_data(payment_method, options = {})
         post_data
       end
 
+      def add_payment_method_card_data_tok(post_data, payment_method)
+        post_data[:payment_method_types] = {}
+        post_data[:payment_method_types][''] = 'card'
```

Comment:
❓ Suggestion:
Just leaving a note here for @curiousepic, is very weird that the post_data method translates a 

`payment_method_types: { '': 'card' }` into `payment_method_types[]=card`

maybe that needs a refactor or a change to make that:  

`payment_method_types:  ['card']` translates into: `payment_method_types[]=card`

 which makes more sense